src/types.rs

```diff
@@ -289,6 +289,12 @@ impl<'prev, 'now> SubmitArgs<'prev, 'now> {
             now: PhantomData,
         }
     }
+
+    #[inline]
+    /// Get the args to pass to raw [`Submitter::enter`](super::Submitter::enter) interface.
+    pub fn get_args(&self) -> &sys::io_uring_getevents_arg {
```

Comment:
We cannot expose private types

Reply:
@quininer how about this ? https://github.com/tokio-rs/io-uring/pull/364